src/pages/Article/ArticlePage.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import PageLayout from '../../components/PageLayout';
+import Button from "@mui/material/Button";
+import { Box } from '@mui/system';
+
+/**
+ * Renders the Article page.
+ * @returns {JSX.Element} ArticlePage component
+ */
+function ArticlePage() {
+  return (
+	<PageLayout viewportPage title='Article'>
+		<Box sx={{ 
+			display: "flex",
+			flexDirection: "column",
+			justifyContent: "center",
+			alignItems: "center",
+			height: "calc(100% - 60px)"
```

Comment:
Why do you reduce the height by 60px?

Reply:
I did that to leave a small space at the bottom which demarcate the container and the Next button.
you would have seen the effect if the container was coloured

Reply:
Why didn't you use a margin for the bottom?

Reply:
there is no specific reason I didn't use it. Margin-bottom can also achieve same purpose, I don't mind changing it

Reply:
but I think the calc function is more suitable to achieve what I wanted